tests/command.bats

```diff
@@ -113,13 +113,13 @@ EOF
     stub pulumi \
          "login : echo 'Logged In'" \
          "config get artifacts --cwd=pulumi/cicd --config-file=/tmp/tmp.000000.yaml --stack=myorg/cicd/production : yq eval '.config.\"cicd:artifacts\"' --output-format=json /tmp/tmp.000000.yaml" \
-         "config set --path artifacts.foo 1.2.2 --cwd=pulumi/cicd --stack=myorg/cicd/production : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.2\"' pulumi/cicd/Pulumi.production.yaml" \
-         "config set --path artifacts.foo 1.2.3 --cwd=pulumi/cicd --stack=myorg/cicd/production : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.3\"' pulumi/cicd/Pulumi.production.yaml" \
-         "config set --path artifacts.bar 4.5.6 --cwd=pulumi/cicd --stack=myorg/cicd/production : yq eval --inplace '.config.\"cicd:artifacts\".bar = \"4.5.6\"' pulumi/cicd/Pulumi.production.yaml" \
+         "config set --path artifacts.[\\\"foo\\\"] 1.2.2 --cwd=pulumi/cicd --stack=myorg/cicd/production : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.2\"' pulumi/cicd/Pulumi.production.yaml" \
+         "config set --path artifacts.[\\\"foo\\\"] 1.2.3 --cwd=pulumi/cicd --stack=myorg/cicd/production : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.3\"' pulumi/cicd/Pulumi.production.yaml" \
+         "config set --path artifacts.[\\\"bar\\\"] 4.5.6 --cwd=pulumi/cicd --stack=myorg/cicd/production : yq eval --inplace '.config.\"cicd:artifacts\".bar = \"4.5.6\"' pulumi/cicd/Pulumi.production.yaml" \
          "config get artifacts --cwd=pulumi/cicd --config-file=/tmp/tmp.000001.yaml --stack=myorg/cicd/testing : yq eval '.config.\"cicd:artifacts\"' --output-format=json /tmp/tmp.000001.yaml" \
-         "config set --path artifacts.foo 1.2.2 --cwd=pulumi/cicd --stack=myorg/cicd/testing : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.2\"' pulumi/cicd/Pulumi.testing.yaml" \
-         "config set --path artifacts.foo 1.2.3 --cwd=pulumi/cicd --stack=myorg/cicd/testing : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.3\"' pulumi/cicd/Pulumi.testing.yaml" \
-         "config set --path artifacts.bar 4.5.6 --cwd=pulumi/cicd --stack=myorg/cicd/testing : yq eval --inplace '.config.\"cicd:artifacts\".bar = \"4.5.6\"' pulumi/cicd/Pulumi.testing.yaml"
+         "config set --path artifacts.[\\\"foo\\\"] 1.2.2 --cwd=pulumi/cicd --stack=myorg/cicd/testing : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.2\"' pulumi/cicd/Pulumi.testing.yaml" \
+         "config set --path artifacts.[\\\"foo\\\"] 1.2.3 --cwd=pulumi/cicd --stack=myorg/cicd/testing : yq eval --inplace '.config.\"cicd:artifacts\".foo = \"1.2.3\"' pulumi/cicd/Pulumi.testing.yaml" \
+         "config set --path artifacts.[\\\"bar\\\"] 4.5.6 --cwd=pulumi/cicd --stack=myorg/cicd/testing : yq eval --inplace '.config.\"cicd:artifacts\".bar = \"4.5.6\"' pulumi/cicd/Pulumi.testing.yaml"
```

Comment:
Just to be a completist, it would be nice to see this end-to-end test pass with an artifact `with.dots.in.the.name`.

Reply:
Man, I'm trying, but BATS debugging is not fun.
bats-mock(pulumi): match failed at idx 0, expected '''', got 'config'

Reply:
Yeah... I ran into this error myself the other day; I think it's from not having spaces around the `:` that separates the stubbed invocation from the "implementation" command. 

That is, it should look like this:

```bash
stub foo "blah blah : echo 'invoked `foo blah blah`'"
```

and not this:
```bash
stub foo "blah blah: echo 'invoked `foo blah blah`'"
```

Not a very fun debugging process for sure 😦